codegen/src/sbt-test/generator/e2e/src/test/scala/twitch/twirp/example/haberdasher/HaberdasherSpec.scala

```diff
@@ -1,4 +1,4 @@
-package twitch.twirp.example.haberdasher.haberdasher
+package twitch.twirp.example.haberdasher
```

Comment:
this will be a breaking change. Projects using Twinagle need to either fix the imports of generated code, or set `scalapbCodeGeneratorOptions := Seq.empty` in `build.sbt`.